packages/aws-cdk-lib/aws-rds/lib/proxy.ts

```diff
@@ -259,6 +281,13 @@ export interface DatabaseProxyOptions {
    * The VPC to associate with the new proxy.
    */
   readonly vpc: ec2.IVpc;
+
+  /**
+   * Specifies the details of authentication used by a proxy to log in as a specific database user.
+   *
+   * @default - The default client password authentication type for the database engine is used.
```

Comment:
Minor nit: this `@default` tag doesn't provide any useful information. is there a link to a doc that shows what the defaults are for different database engines?

Reply:
I wasn't able to find one.
I agree that this is unclear, but I'm not sure how we should state this.
Should we just say that CloudFormation handles the default given the specified engine?

Reply:
Yeah, lest just say CloudFormation handles the default. Should be clear that its not us making the decisions.